proto/store/setting.proto

```diff
@@ -261,10 +261,23 @@ message MaskingAlgorithmSetting {
       string salt = 1;
     }
 
+    message InnerOuterMask {
+      int32 prefix_len = 1;
+      int32 suffix_len = 2;
+
+      enum InnerOuterMaskType {
+        INNER = 0;
```

Comment:
Please follow our proto enum format, the first enum should be `{name in upper case}_UNSPECIFIED = 0;`
BTW I'd like to use `MaskType` instead of the `InnerOuterMaskType` (because the enum is already wrapped in the `InnerOuterMask` message)

For example,

```
enum MaskType {
  MASK_TYPE_UNSPECIFIED = 0;
  INNER = 1;
  OUTER = 2;
}
```